korman/nodes/node_responder.py

```diff
@@ -227,40 +226,39 @@ def _set_default_state(self, value):
                                  set=_set_default_state,
                                  options=set())
 
-    input_sockets = OrderedDict([
-        ("condition", {
+    input_sockets: dict[str, any] = {
+        "condition": {
             "text": "Triggers State",
             "type": "PlasmaRespStateSocket",
             "spawn_empty": True,
-        }),
-        ("resp", {
+        },
+        "resp": {
             "text": "Responder",
             "type": "PlasmaRespStateRefSocket",
             "valid_link_nodes": "PlasmaResponderNode",
             "valid_link_sockets": "PlasmaRespStateRefSocket",
-        }),
-    ])
+        },
+    }
 
-    output_sockets = OrderedDict([
-        # This socket has been deprecated.
-        ("cmds", {
+    output_sockets = {
+        # While this is deprecated I might aswell also convert it.
```

Comment:
Unsure if this comment should be removed/reworded, I'll leave it to Hoikas to make that decision

Reply:
Yeah, let's leave this in. Korman still has a policy of trying to upgrade in place as much user data as possible.